pages/faq/troubleshooting/faq.md

```diff
@@ -30,6 +31,12 @@ __Note:__ If you do not see an option to choose a starter or microservices appli
 
 If you are running a Docker-in-Docker setup, which builds a single application container on the resin.io servers but has a `docker-compose.yml` file at the root of the project, you'll want to rename the file to something like `dind-compose.yml`. Then when you run Docker Compose in your container, you can use the `-f` flag with the new file name: `docker-compose -f dind-compose.yml up`.
 
+##### Can I mix device types in an application?
+
+It is possible to have devices of different types in the same application, as long as the devices share the same architecture. For example, you could have an application with both Raspberry Pi 3 and BeagleBone Black devices, as they both use an ARMv7 processor. However, you could not have any Intel NUC devices as part of the same application, as those devices have x86-64 processors.
```

Comment:
Perhaps it's worth adding a link to https://docs.resin.io/reference/base-images/devicetypes/ ? (which shows which architectures different devices use)